<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>imchenwen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imchenwen.qrc">
    <normaloff>:/res/imchenwen32.png</normaloff>:/res/imchenwen32.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="mainHorizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>3</number>
      </property>
      <widget class="QTreeView" name="navigatorTreeView"/>
      <widget class="QTableView" name="resourceListView"/>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
          <item>
           <widget class="QLineEdit" name="urlEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="goButton">
            <property name="toolTip">
             <string>Parse the URL and invoke external media player</string>
            </property>
            <property name="statusTip">
             <string>Parse the URL and invoke external media player</string>
            </property>
            <property name="text">
             <string>Go!</string>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Ctrl+G</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="playListView"/>
        </item>
        <item>
         <widget class="QTableView" name="playedListView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Tool">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
    <addaction name="actionOption"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContent"/>
    <addaction name="actionHomepage"/>
    <addaction name="actionBuy"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tool"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Standard</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="text">
    <string>&amp;Option...</string>
   </property>
  </action>
  <action name="actionContent">
   <property name="text">
    <string>&amp;Content...</string>
   </property>
  </action>
  <action name="actionHomepage">
   <property name="text">
    <string>Homepage...</string>
   </property>
  </action>
  <action name="actionBuy">
   <property name="text">
    <string>&amp;Buy...</string>
   </property>
   <property name="iconText">
    <string>Buy</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="imchenwen.qrc">
     <normaloff>:/res/imchenwen32.png</normaloff>:/res/imchenwen32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imchenwen.qrc"/>
 </resources>
 <connections/>
</ui>
